src/utils/WeakMap.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable class-methods-use-this */
```

Comment:
@matthieuprat sorry for build failure, `yarn test` was passing, now I'm running `yarn lint && yarn test`